plugins/plugin-kiali/src/components/panel/Graph.tsx

```diff
@@ -0,0 +1,300 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import React, { memo, useCallback, useEffect, useRef, useState } from 'react';
+import CytoscapeComponent from 'react-cytoscapejs';
+import cytoscape from 'cytoscape';
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+import dagre from 'cytoscape-dagre';
+
+import { IEdgeWrapper, INodeData, INodeWrapper } from '../../utils/interfaces';
+import { IPluginInstance, ITimes } from '@kobsio/shared';
+import Edge from './details/Edge';
+import Node from './details/Node';
+
+cytoscape.use(dagre);
```

Comment:
And here you can register the import:
```suggestion
cytoscape.use(dagre);
nodeHtmlLabel(cytoscape);
```

If this is missing I get an `undefined is not a function (near '...cy.nodeHtmlLabel...')` error.